torch/_dynamo/variables/torch_function.py

```diff
@@ -632,9 +638,17 @@ def var_getattr(self, tx: "InstructionTranslator", name):
         # base tensors, custom attribute accesses will graph break.
         import torch
 
+        # I think only `_base` is breaking because we aren't modelling view
+        # relationship perfectly in some scenarios.
         if name in banned_attrs:
-            unimplemented(
-                f"Accessing {name} on a tensor subclass with a __torch_function__ override is not supported"
+            unimplemented_v2(
+                gb_type="Tensor subclass attribute access",
+                context=f"{name}",
+                explanation="`torch.compile` currently can't trace this",
```

Comment:
Any specific reason why this is not supported (i.e. why it's in banned_attrs)?

Reply:
Not sure, probably just plumbing-needed and low-priority.